Resources/views/Layouts/macros.html.twig

```diff
@@ -103,13 +121,13 @@
 {% endmacro %}
 
 {# params string colorAltern    how to style timegrid, color direction, line or column #}
-{% macro calendar(dom_id, stopPointLevel, externalNetworkId, stopTimetable, layout, block, calendar, notes, hours_range, linesMin, notesType, colorAltern = 'line') %}
+{% macro calendar(dom_id, externalNetworkId, stopTimetable, layout, block, calendar, notes, hours_range, linesMin, notesType, colorAltern = 'line') %}
```

Comment:
You deleted the parametre "stopPointLevel" there, but this parametre is not deleted in the file "/Resources/views/Layouts/timegrid.html.twig".